devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy

```diff
@@ -27,6 +27,7 @@ metadata {
         fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0702"
         fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0702, 0B05", outClusters: "0003, 000A, 0019", manufacturer: "Jasco Products", model: "45853", deviceJoinName: "GE ZigBee Plug-In Switch"
         fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0702, 0B05", outClusters: "000A, 0019", manufacturer: "Jasco Products", model: "45856", deviceJoinName: "GE ZigBee In-Wall Switch"
+        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0702", manufacturer: "ClimaxTechnology", model: "PSM_00.00.00.35TC", deviceJoinName: "Ozom Smart Plug"
```

Comment:
Correct me if I'm wrong but I think this device also has a single out cluster `0000`. If that's true can you add `outClusters: "0000"` to the fingerprint?

Reply:
@tpmanley Maybe we should make it a standard to add RAW fingerprint as a comment?

Reply:
@tpmanley , you are right. I added outClusters: "0000" to fingerprint.
@MAblewiczS, I added RAW fingerprint as comment.
Can you check please?